### ISSUE TITLE
Facebook notification sounds interrupting music 

### ISSUE THREAD

**Issue by Gilligan94** [SoundControl/Issues#125]
I think there needs to be an exclusion list of some kind now that HTML5 audio is an option. Notifications sounds cut through music and if I pause something and then get a Facebook notification when I hit play again it'll just play the notification sound

**Comment by SoundControl** [comment]
Yep, that definitely needs to be improved. Thanks for the report

**Comment by SoundControl** [comment]
For now, web audio less than 4 seconds won't be handled by SoundControl

**Comment by SoundControl** [comment]
"""
Recently I experienced an odd issue with sound control plus.  (in Windows 7 64 bit, using chrome ver 35 beta.)

I had no open players, and started receiving Vine notifications, apparently through sound control.  They were all notifications of vine posts from one particular user, and the sound control icon changed to match the notification icon, the same way it would if I had Pandora or another music service open. As I am not a Vine user, or had anything open that was remotely vine related, I am a little baffled.  
Unfortunately I did not think to do a screen capture at the time, so this will have to be a bizarre anecdotal report.  

The only thing I can think of that would trigger this would be an unnoticed comments box with someone posting embedded vine content, maybe?
"""

Sounds like it's not completely fixed
